client/logs_test.go

```diff
@@ -111,6 +116,38 @@ func (cs *clientSuite) TestLogsAll(c *check.C) {
 `[1:])
 }
 
+func (cs *clientSuite) TestLogsLong(c *check.C) {
+	shortLog := `{"time":"2021-05-03T03:55:49.360994155Z","service":"thing","message":"log 1\n"}`
+	var longLog strings.Builder
+	longLog.WriteString(`{"time":"2021-05-03T03:55:49.460994155Z","service":"verbose","message":"`)
+	for i := 0; i < maxMessageSize; i++ {
+		longLog.WriteString("a")
+	}
+	longLog.WriteString(`\n"}`)
```

Comment:
No need for a `strings.Builder` and loop here, you can just use `strings.Repeat`:

```go
longLog := `{..., "message":"` + strings.Repeat("a", maxMessageSize) + `\n"}`
```